### PR TITLE
[Method References] LPS-162758 Convert method references to field references in module portal-search-elasticsearch7-impl | part 2

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I convert method references to field references in module portal-search-elasticsearch7-impl.

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by catarineaguiar** [comment]
no regressions.

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/document/ElasticsearchDocumentRequestExecutorTest.java [hunk 1]
```diff
@@ -42,21 +43,30 @@ public class ElasticsearchDocumentRequestExecutorTest {
...
-						_deleteDocumentRequestExecutor);
-					setIndexDocumentRequestExecutor(
-						_indexDocumentRequestExecutor);
-					setUpdateByQueryDocumentRequestExecutor(
-						_updateByQueryDocumentRequestExecutor);
-					setUpdateDocumentRequestExecutor(
-						_updateDocumentRequestExecutor);
-				}
-			};
+			elasticsearchDocumentRequestExecutor;
```
Comment (dantewang) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/document/ElasticsearchDocumentRequestExecutorTest.java:69]: @W2jx Please avoid this if possible, see my comments from the other PR.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/document/ElasticsearchDocumentRequestExecutorTest.java (hunk 1 above)
Comment (W2jx) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/search/engine/adapter/document/ElasticsearchDocumentRequestExecutorTest.java:69]: I will fix it.

**Comment by W2jx** [comment]
Force-Push is just to rebase and fix the problems mentioned in comments.

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
Hi @catarineaguiar , Could you please help me check the test results ? Thanks!

**Comment by catarineaguiar** [comment]
@W2jx no regressions.

**Comment by dantewang** [comment]
@shuyangzhou @tinatian Forwarding this one as all tests passed review and changes are direct replacements.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/123675

**Pull request closed without merging**